sys-devel/gcc/gcc-13.3.0_2023_08_10.recipe

```diff
@@ -79,7 +79,8 @@ PROVIDES_fortran="
 	"
 REQUIRES_fortran="
 	haiku$secondaryArchSuffix
-	gcc${secondaryArchSuffix}_syslibs == $portVersion base
+	gcc${secondaryArchSuffix} == $portVersion base
```

Comment:
The braces are unneeded in this case (but don't harm either).

Reply:
@jmairboeck these are 2 different packages? (see _syslibs) where those are required for syslibs?
Or if the case is that it doesn't require syslibs the {} can be left out for gcc.

Reply:
For syslibs (the next line), the braces are required because of the following `_` (just like the deleted line above), but this line could be

```suggestion
	gcc$secondaryArchSuffix == $portVersion base
```

for simplicity.